diff -urN nosefart-2.7-mls/Makefile nosefart-2.7-mls-patched/Makefile
--- nosefart-2.7-mls/Makefile	2007-12-14 07:46:22.000000000 +0300
+++ nosefart-2.7-mls-patched/Makefile	2011-07-31 18:42:30.000000000 +0400
@@ -61,7 +61,7 @@
 
 ALL_OBJECTS = $(OBJECTS)
 
-ALL_TARGETS = $(BUILDTOP)/$(NAME)
+ALL_TARGETS = $(BUILDTOP)/$(NAME)
 
 ################################
 # Rules
@@ -104,6 +104,8 @@
 ################################
 # The real heavy lifting
 
+$(OBJECTS): $(BUILDDIR) $(BUILDTOP)/config.h
+
 $(BUILDTOP)/$(NAME): $(OBJECTS)
 	$(CC) $(NSFINFO_CFLAGS) $(LDFLAGS) -o $@ $^
 
